catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -104,6 +111,25 @@ private void handleTreValues(Metacard metacard, NitfAttribute attribute, Tre tre
   private Attribute populateAttribute(Metacard metacard, String attributeName, Serializable value) {
     Attribute currentAttribute = metacard.getAttribute(attributeName);
 
+    // check if country code
+    if (COUNTRY_CODE_ATTRIBUTES.contains(attributeName)) {
```

Comment:
❓  Should we add test coverage for this?

Reply:
Yep I will eventually

Reply:
❗️  When you say eventually, do you mean as part of this same ticket? If not then I have to disagree, test cases should be added here. We shouldn't be pushing code without adding test cases and thus increasing tech debt.